scripts/build_pytorch_android.sh

```diff
@@ -59,9 +59,18 @@ INCLUDE_DIR=$PYTORCH_ANDROID_DIR/pytorch_android/src/main/cpp/libtorch_include
 mkdir -p $LIB_DIR
 mkdir -p $INCLUDE_DIR
 
-for abi in $(echo $ABIS_LIST | tr ',' '\n')
+IFS=', ' read -r -a abis_array <<< "$ABIS_LIST"
+for abi in arm64-v8a armeabi-v7a x86_64 x86
+do
+  if [[ ! ${abis_array[*]} =~ $abi ]]
+  then
+    rm -f $LIB_DIR/$abi
+    rm -f $INCLUDE_DIR/$abi
+  fi
+done
```

Comment:
I think this is to address broken symbolic links left from previous build, right? I just updated this logic recently - I simply cleared $LIB_DIR / $INCLUDE_DIR at each rebuild as these directories only contain symbolic links (super cheap to recreate).